lively.components/youtube-morph.cp.js

```diff
@@ -1,6 +1,15 @@
 import { component, ViewModel, HTMLMorph } from 'lively.morphic';
 import { pt } from 'lively.graphics/geometry-2d.js';
 
+// Browsers do not like us importing this in more than once place...
+if (!window.liteYouTubeEmbed && !window.liteYouteTubeLoading) {
+  window.liteYoutubeLoading = true;
+  System.import('lite-youtube-embed').then((embed) => {
+    window.liteYouTubeEmbed = embed;
+    delete window.liteYoutubeLoading;
```

Comment:
We have this nice function `fun.guardNamed(...)`. I think it even supports asynchronous functions as callback, that way we can save ourselves the weird `then()` stuff.

```suggestion
      fun.guardNamed('import-lite-youtube-embed', async () => {
           window.liteYoutubeEmbed = await System.import('lite-youtube-embed');
       })();
```